app/screens/Settings.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { View, StyleSheet, SafeAreaView } from 'react-native'
+import MedicBleepLogo from '../components/MedicBleepLogo'
+import { SettingsCell } from '../components'
+import { colors } from '../config'
+
+const { white } = colors
+
+const Settings = ({ started }) => {
+  const storeHistoryValueChanged = (value) => {}
+  const shiftReminderValueChanged = (value) => {}
+  const aboutMbCellPressed = () => {}
+  const followCellPressed = () => {}
+
+  const { container, contentWrapper } = styles
+  return (
+    <SafeAreaView style={container}>
+      <View style={contentWrapper}>
+        <SettingsCell
+          title={'Store History'}
+          onCellPress={null}
+          onSwitchValueChange={storeHistoryValueChanged}
+          titleNumLines={1}
+          iconName={null}
+        />
+        <SettingsCell
+          title={'Two Hour On Shift Reminders'}
+          onCellPress={null}
+          onSwitchValueChange={shiftReminderValueChanged}
+          titleNumLines={2}
+          iconName={null}
+        />
+        <SettingsCell
+          title={'About Medic Bleep'}
+          onCellPress={aboutMbCellPressed}
+          onSwitchValueChange={null}
+          titleNumLines={1}
+          iconName={'chevron-right'}
+        />
+        <SettingsCell
+          title={'Follow @medicbleep on Twitter'}
+          onCellPress={followCellPressed}
+          onSwitchValueChange={null}
+          titleNumLines={2}
+          iconName={'chevron-right'}
+        />
```

Comment:
I think its fine to lay it out like this, no need to complicated it with an array of values.

Reply:
One thing we need to do here is the actions. The first two cells need redux, you'll fire an action and toggle the values.

For the bottom two we can use https://reactnative.dev/docs/linking